templates/nunjucks/_base.njk.html

```diff
@@ -3,9 +3,9 @@
 {% block pageTitle %}{% if page.errorList %}Error: {% endif %}{% if page.heading %}{{ page.heading }}{% else %}{% if page.components[0].label.html %}{{ page.components[0].label.html }}{% else %}{{ page.components[0].legend.html }}{% endif %}{% endif %}{% if page._id != 'page.start' %} - {{ page.header.serviceName }}{% endif %} - GOV.UK{% endblock %}
 
 {% block head %}
-<link rel="stylesheet" href="/assets/govuk-frontend-{{govuk_frontend_version}}.min.css">
+<link rel="stylesheet" href="/assets/govuk-frontend-3.3.0.min.css">
```

Comment:
These are explicit references to files contained in the `assets` folder at the root of this project

Other files (like fonts and icons) will be served from that folder too

Reply:
(They are manually added from the GOV.UK Frontend repository on GitHub)